modules/netns-isolation.nix

```diff
@@ -297,6 +299,7 @@ in {
         id = 31;
         connections = [ "bitcoind" ];
       };
+      # id = 32 reserved for mempool
```

Comment:
How about "mempool" -> "mempool extension flake" to clarify that mempool isn't part of this code base?

Wouldn't it be better to explicitly reserve a different range for extension flakes, e.g., anything > 1000?

Reply:
Mempool will be added as a nix-bitcoin module when the backend bug is resolved.

New extension flakes will be assigned `id`s >= 100. (`id`s map to the last byte of a IPv4 address.)

Reply:
Done.

New extension flakes will be assigned `id`s >= 100. (`id`s map to the last byte of a IPv4 address.)